2.4/test/run

```diff
@@ -11,85 +11,22 @@ shopt -s nullglob
 
 IMAGE_NAME=${IMAGE_NAME-openshift/mongodb-24-centos7-candidate}
 
-CIDFILE_DIR=$(mktemp --suffix=mongodb_test_cidfiles -d)
+source ../common/test-lib.sh
+source ../test/lib
```

Comment:
This needs to become a symlink instead, so that we can make sure the tests do not fail even when run in isolation.